src/Microsoft.Health.Fhir.Core/Features/Search/Parameters/ISearchParameterOperations.cs

```diff
@@ -12,11 +12,11 @@ namespace Microsoft.Health.Fhir.Core.Features.Search.Parameters
 {
     public interface ISearchParameterOperations
     {
-        Task AddSearchParameterAsync(ITypedElement searchParam);
+        Task AddSearchParameterAsync(ITypedElement searchParam, CancellationToken cancellationToken = default);
```

Comment:
In general we avoid using default values with ContinuationToken, this helps ensure that the calling code is doing the right thing and not accidentally missing it

Reply:
@brendankowitz  If the preference is to break when it isn't passed in then that is fine. Would you like me to switch to throwing an argument null exception if that value isn't passed instead of going for a default value? 

There are other portions of the code that also utilize a default cancellation token and that is typically what I do to prevent the system from breaking when a new parameter is introduced. So for consistency would those need to be removed?

Reply:
To summarize our discussion: we will remove the default cancellation token as an optional argument so that it can be controlled from the top of the call stack (the FHIR controller).